runtime/config-deployer-service/ballerina/DeployerClient.bal

```diff
@@ -59,7 +59,8 @@ public class DeployerClient {
                 _ = check self.deleteBackends(api, organization);
                 _ = check self.deleteAPIPolicyCRs(api, organization);
                 _ = check self.deleteInterceptorServiceCRs(api, organization);
-                return {body: "API with id " + apiId + " undeployed successfully"};
+                json response = {status: "API with id " + apiId + " undeployed successfully"};
```

Comment:
Lets use string template expression to construct the string

Reply:
resolved